csrc/host_ir/lower.h

```diff
@@ -35,6 +36,12 @@ class HostIrLower {
       std::unique_ptr<Fusion> fusion,
       DeviceIdxType my_device_index);
 
+  static bool isLoweredAsStandaloneHostOp(Expr* expr);
+
+  static bool ShouldMergeSegmentedGroups(
+      SegmentedGroup* group1,
+      SegmentedGroup* group2);
```

Comment:
This function could not been exposed if it wasn't used in`tests/cpp/test_resharding.cpp`. We could alternatively not expose it and just duplicate its implementation (which is short and simple) in the test.